db/indexOld.js

```diff
@@ -0,0 +1,69 @@
+const mongoose = require('mongoose');
```

Comment:
Assuming this is just renamed to use index.js for your postgres db.

Reply:
Yes, just in case I want to refer back to the original quickly. I'll delete it eventually.